.circleci/config.yml

```
@@ -39,10 +46,10 @@ workflows:
             tags:
               only: /[0-9]+(\.[0-9]+)*(-.*)*/
       - build-image:
+          context: falco
           requires:
             - test
             - lint
           filters:
             tags:
               only: /[0-9]+(\.[0-9]+)*(-.*)*/
```

```suggestion
            branches:
              only: "master"
```
If we want to push the `lastest` tag only when the master has been updated, I think we have to filter just for that

---

we also want to just to build in PRs to make sure the image can be built

---

So we want to build on every branch, but push the image for the master only?

---

we go with two jobs as discussed in slack